mipengine/node/tasks/smpc.py

```diff
@@ -172,4 +172,8 @@ def _create_smpc_results_table(
 
 def _get_smpc_values_from_table_data(table_data: List[ColumnData]):
     node_id_column, values_column = table_data
+
+    if not values_column.data:
+        raise SMPCUsageError("A node doesn't have data to contribute to the SMPC.")
```

Comment:
I don't think that's it's a good idea to "hide" this logic inside the SMPC functionality.

The approach we currently take is to handle this inside the algorithm, you can check anova one way.

It's also not safe to add "neutral" values because if no node has any values then the "neutral" values will be returned as a result.